scripts/utils.py

```diff
@@ -1,7 +1,12 @@
 # -*- coding: utf-8 -*-
+import os
 from collections import OrderedDict
+
 from git import Repo
-import os
+
+# Languages with insufficient translation data are excluded
+# TODO: Automate with exclusion criteria.
```

Comment:
:lipstick: Do you plan to do this a part of this pull request? Otherwise, it may be better to create a Github issue for it instead and remove this line.

Reply:
The plan was to merge the script together with freshly pulled data from CLDR, but there are too many changes there. I'm postponing that to later than 0.7.2.